### DISCUSSION TITLE
A noob in python and stuff

### DISCUSSION THREAD

**Discussion by Fushii77** [TiagoMarinho/Soph#30] [Q&A]
I am a real noob at all that kind of stuff but i wanted to make the bot for a discord server between friend : 
- Where do i execute the run command ?
- Where do i put the --api ?
Thanks if you can help ^^

**Comment by TiagoMarinho** [comment]
Both `npm run deploy` and `node .` commands should be executed inside Soph's root folder. In other words, inside the folder that was created after you ran `git clone https://github.com/TiagoMarinho/Soph`.

`--api` should be added to the `webui-user.bat` file in AUTOMATIC1111's Stable Diffusion WebUI.

Note that Python is not used in this project, the bot is written in Javascript. Anyway, if you have any other questions, please don't hesitate to ask.
